rust-toolchain.toml

```diff
@@ -1,5 +1,5 @@
 [toolchain]
-channel = "nightly-2023-09-05"
+channel = "nightly-2023-10-31"
```

Comment:
We are unable to update to the latest version due to an issue with fuzzer coverage.